alg/crc32c.c

```diff
@@ -63,7 +63,7 @@ init(void)
 
 	/* Fill in tables. */
 	for (i = 0; i < 256; i++) {
-		r = reverse(i);
+		r = reverse(i & 0xffff);
```

Comment:
erm, yes, well, as a cellist, counting isn't my strong point.  :)

Curious, though: elsewhere, you've preferred `& 0xff` instead of a cast.  Is that because this is 32 bits, or because it's not in the crypto directory, or...?